addons/udes_report/tests/test_stock_export.py

```diff
@@ -43,9 +55,9 @@ def test_stock_no_locations(self):
     def test_stock_all_excluded_locations(self):
         ''' Stock - should error in case all locations are excluded '''
         self.stock_export.included_locations = \
-            self.LocationObj.browse(self.stock_location)
+            self.LocationObj.browse(self.stock_location.ids)
```

Comment:
Browse is not needed, lines 57 and 58 are like:
self.stock_export.included_locations = self.stock_location

This might apply to excluded_locations and other tests.